README.md

```diff
@@ -9,7 +9,9 @@ Programming language that compiles to Bash. It's a high level programming langua
 > [!Warning]
 > This software is not ready for extended usage.
 
-[Join our Discord!](https://discord.com/invite/cjHjxbsDvZ)
+[Join our Discord](https://discord.com/invite/cjHjxbsDvZ) or
```

Comment:
the various channels are synced?

Reply:
It doesn't seem to be synced

Reply:
So who follows the matrix channels if they are not synced with discord?

Reply:
Not sure 🤷‍♂️

Reply:
So I don't think that we can merge if we have no control of that rooms, idially they should be synced with the general room on discord or we have to reject the PR.

Reply:
@Mte90 this could live it's own life in my opinion. We can remove the url if the community there isn't active anymore in a month from now

Reply:
If we mention in the readme it should be moderate otherwise is unofficial ffor the project so doesn't deserve a mention, especially as it is new.

Reply:
See the conversation at https://github.com/Ph0enixKM/Amber/discussions/241#discussioncomment-9903862